Libraries/Components/ViewPager/ViewPagerAndroid.macos.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
```

Comment:
Why no @flow check for this file?

Reply:
ViewPagerAndroid.ios.js is the same: no `@flow` and if added it fails.

Reply:
I was more wondering if it's a failure we should fix

Reply:
The facebook ViewPagerAndroid.ios.js file also has no @flow.   If added it fails to validate.   We're just matching ios.